File: modulo1/segmento3/apunte7Funciones/ejercicio3.py
def maximo(num1,num2,num3):
	
	if num1>=num2:
		maximo= num1
	else:
		maximo=num2
	
	if num3>= maximo:
		maximo=num3
	else:
		maximo=maximo
	
	return maximo

num1=int(input("Ingrese un numero: " ))
num2=int(input("Ingrese un numero: " ))
num3=int(input("Ingrese un numero: " ))

respuesta= maximo(num1,num2,num3)
print("Maximo: "+  str(respuesta))
		

File: modulo1/segmento3/apunte7Funciones/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3{
	public static String multiplos(Integer num1,Integer num2){
		String respuesta="";
		if(num1%num2==0){
			respuesta=num1 + " es multiplo de " + num2;
		}else if(num2%num1==0){
			respuesta=num2 + " es multiplo de " + num1;
		}else{
			respuesta= "No son multiplos";
		
		}
		return respuesta;
	}
	
	public static void main(String args[]){
		Scanner teclado= new Scanner(System.in);
		String respuesta="";

		System.out.println("Ingrese un numero: ");
		Integer num1= Integer.parseInt(teclado.nextLine());

		System.out.println("Ingrese un numero: ");
		Integer num2= Integer.parseInt(teclado.nextLine());

		respuesta= multiplos(num1,num2);
		System.out.println(respuesta);
	}
}
